plugin-server/src/worker/ingestion/action-matcher.ts

```diff
@@ -137,16 +138,55 @@ export class ActionMatcher {
         this.statsd = statsd
     }
 
-    /** Get all actions matched to the event. */
     public async match(event: PostIngestionEvent, elements?: Element[]): Promise<Action[]> {
+        const legacyResponse = await this.matchLegacy(event, elements)
+        const hogVMResponse = this.matchBytecode(event, elements)
+
+        if (
+            legacyResponse.length !== hogVMResponse.length ||
+            !legacyResponse.every((action, index) => action.id == hogVMResponse[index]?.id)
+        ) {
+            this.statsd?.increment('action_matches_hogvm_legacy_mismatch')
+        } else {
+            this.statsd?.increment('action_matches_hogvm_legacy_match')
+        }
+        console.log({ hogVM: hogVMResponse.map((a) => a?.id), legacy: legacyResponse.map((a) => a?.id) })
+
+        return legacyResponse
+    }
+
+    /** Get all actions matched to the event. */
+    public matchBytecode(event: PostIngestionEvent, elements?: Element[]): (Action | undefined)[] {
+        const matchingStart = new Date()
```

Comment:
Let's use `performance.now()` instead of `new Date()`. We'll need to pass in `performance.now() - matchingStart` instead of just `matchingStart`, but `performance` is more reliable for metrics